homeassistant/components/intellidrive/const.py

```diff
@@ -0,0 +1,7 @@
+"""Constants for the Intellidrive integration."""
+
+DOMAIN = "intellidrive"
+UPDATE_INTERVAL = 3
```

Comment:
This seems a bit on the short side, is it really needed to refresh every 3s?

Reply:
Yeah thats possible, I increased to 5